### PR TITLE
SONAR: Immediately return this expression instead of assigning it to the tem…

### PR DESCRIPTION
**Pull request by slawekk87**
…porary variable pt2

## Changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

### Final checklist

- [ ] My code is written according to the guidelines found [here] (https://yetiforce.com/en/knowledge-base/documentation/developer-documentation/item/how-to-create-php-files).
- [ ] I have signed the Contributor Licence Agreement between me and YetiForce.

### CODE REVIEW

File: modules/Settings/Vtiger/models/Menu.php [hunk 1]
```diff
@@ -55,9 +55,7 @@ public function getUrl()
 	{
 		$url = $this->get('linkto');
 		$url = App\Purifier::decodeHtml($url);
-		$url .= '&block=' . $this->getId();
-
-		return $url;
+		return $url . '&block=' . $this->getId();
```
Comment (TomaszQr) [inline modules/Settings/Vtiger/models/Menu.php]: return  App\Purifier::decodeHtml($this->get('linkto')) . '&block=' . $this->getId();

File: modules/Vtiger/models/InventoryField.php [hunk 2]
```diff
@@ -39,9 +39,7 @@ public function getTableName($type = 'data')
 		}
 		$focus = CRMEntity::getInstance($this->get('module'));
 		$basetable = $focus->table_name;
-		$supfield = $basetable . $prefix;
-
-		return $supfield;
+		return $basetable . $prefix;
```
Comment (TomaszQr) [inline modules/Vtiger/models/InventoryField.php]: return CRMEntity::getInstance($this->get('module'))->table_name . $prefix;

File: modules/Vtiger/models/Link.php [hunk 3]
```diff
@@ -305,9 +303,7 @@ public function convertToNativeLink()
 
 		$url = implode('&', $parametersParts);
 		//To convert single quotes and double quotes
-		$url = \App\Purifier::encodeHtml($url);
-
-		return $url;
+		return \App\Purifier::encodeHtml($url);
```
Comment (TomaszQr) [inline modules/Vtiger/models/Link.php]: return \App\Purifier::encodeHtml(implode('&', $parametersParts)); ? test it

File: modules/Vtiger/uitypes/Reminder.php [hunk 4]
```diff
@@ -39,9 +39,7 @@ public function getEditViewDisplayValue($value, $recordModel = false)
 			$rem_days = floor($value / (24 * 60));
 			$rem_hrs = floor(($value - $rem_days * 24 * 60) / 60);
 			$rem_min = ($value - ($rem_days * 24 * 60)) % 60;
-			$reminder_time = [$rem_days, $rem_hrs, $rem_min];
-
-			return $reminder_time;
+			return [$rem_days, $rem_hrs, $rem_min];
```
Comment (TomaszQr) [inline modules/Vtiger/uitypes/Reminder.php]: improve name of variables

File: modules/Vtiger/models/ChartFilter.php [hunk 5]
```diff
@@ -741,8 +741,7 @@ protected function getQuery($filter)
 		}
 		$query = $queryGenerator->createQuery();
 		// we want colors from picklists if available
```
Comment (rskrzypczak) [inline modules/Vtiger/models/ChartFilter.php]: remove comment

File: modules/Vtiger/models/Field.php [hunk 6]
```diff
@@ -1412,21 +1410,21 @@ public function getRangeValues()
 				} else {
 					return '-2147483648,2147483647';
 				}
-				// no break
+			// no break
```
Comment (rskrzypczak) [inline modules/Vtiger/models/Field.php]: ?

File: modules/Vtiger/models/InventoryField.php [hunk 7]
```diff
@@ -531,8 +527,8 @@ public function delete($param)
 	 */
 	public function getUniqueID($instance)
 	{
-		return (int) (new \App\Db\Query())->from($this->getTableName('fields'))->where(['invtype' => $instance->getName()])
-			->max('id') + 1;
+		return (int)(new \App\Db\Query())->from($this->getTableName('fields'))->where(['invtype' => $instance->getName()])
```
Comment (rskrzypczak) [inline modules/Vtiger/models/InventoryField.php]: php-fixer

**Pull request merged**